### PR TITLE
General improvements to twist_controller and waypoint_updater

### PR DESCRIPTION
**Pull request by vguerra**
Mainly improvements to documentation and removing unused variables.

#82

### CODE REVIEW

File: ros/src/twist_controller/dbw_node.py [hunk 1]
```diff
@@ -113,6 +120,7 @@ def __init__(self):
         self.loop()
 
     def loop(self):
+        """Loop that computes throttle, brake and steer to publish."""
         rate = rospy.Rate(10)  # originally 50Hz
```
Comment (asimonov) [inline ros/src/twist_controller/dbw_node.py]: do you think we shall go back to 50hz for carla?
dont remember for sure, but I think someone mentioned on slack this one better be 50hz.

File: ros/src/twist_controller/dbw_node.py [hunk 2]
```diff
@@ -186,7 +195,8 @@ def waypoints_cb(self, msg):
             self.waypoints = msg.waypoints
 
     def _valid_state(self):
-        """ Checks whethear node has all information needed to operate correctly
+        """ Checks whethear node has all information needed to operate correctly.
```
Comment (asimonov) [inline ros/src/twist_controller/dbw_node.py]: typo: wether

File: ros/src/twist_controller/twist_controller.py [hunk 3]
```diff
@@ -78,30 +86,30 @@ def control(self,
                                                                 angular_velocity=angular_velocity,
                                                                 current_velocity=current_velocity)
 
-            corrective_steer = self.steer_pid.step(error=cte, sample_time=sample_time)
+            corrective_steer = self.steer_pid.step(
```
Comment (asimonov) [inline ros/src/twist_controller/twist_controller.py:89]: hm, i liked the old formatting :)

File: ros/src/twist_controller/twist_controller.py [hunk 4]
```diff
@@ -78,30 +86,30 @@ def control(self,
...
-                          steer, cte, corrective_steer, predictive_steer)
+                           steer, cte, corrective_steer, predictive_steer)
 
             vel_delta = linear_velocity - current_velocity
-            res_accel = self.accel_pid.step(error=vel_delta, sample_time=sample_time)
-            
-            ###### Filter start
-            res_accel = self.avg_filter.filter(res_accel)
-            ###### Filter End
+            res_accel = self.accel_pid.step(
```
Comment (asimonov) [inline ros/src/twist_controller/twist_controller.py:98]: is it a result of auto formatter?

**Review by asimonov — APPROVED** [review]
looks great!

**Comment by asimonov** [comment]
looks like same Travis issues with Docker build as in previous PR that I created:

https://travis-ci.org/Kairos-Automotive/carla-brain/builds/297192668#L626

File: ros/src/twist_controller/twist_controller.py (hunk 4 above)
Comment (vguerra) [reply ros/src/twist_controller/twist_controller.py:98]: yes.. it's the resultl of autoformater in visual studio code.

File: ros/src/twist_controller/dbw_node.py (hunk 2 above)
Comment (vguerra) [reply ros/src/twist_controller/dbw_node.py]: fixed.

File: ros/src/twist_controller/dbw_node.py (hunk 1 above)
Comment (vguerra) [reply ros/src/twist_controller/dbw_node.py]: yes.. probably is a good idea to go up to 50 hz. Ill try locally.

**Review by yhoazk — APPROVED** [review]
Good clean-up

**Review by avrabe — APPROVED** [review]

**Comment by avrabe** [comment]
@vguerra Just fixed the build error. If you want to retrigger with the changes you'd need to rebase your changes.

**Review by dabavil — APPROVED** [review]

**Pull request merged**